### DISCUSSION TITLE
Can I re-install the elastalert container by itself?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#9396] [Unsupported Versions]
Hi all. when I run security onion, sometimes I get elastalert container is missing. Is there a way I can re install this container without have to create a whole new VM?

   Thanks for any help or advice

**Comment by dougburks** [comment]
You've asked this previously:
https://github.com/Security-Onion-Solutions/securityonion/discussions/8961

You shouldn't need to reinstall the container. You just need to figure out why Elastalert is not running.

Based on https://github.com/Security-Onion-Solutions/securityonion/discussions/8961 and your other discussions asking about DHCP, it's most likely that your Security Onion installation is using DHCP and changing its IP address preventing Elastalert from connecting to Elasticsearch. If that's the case, please make sure you are using a static IP address.
